MIP40/MIP40c3-Subproposals/MIP40c3-SPX.md

```diff
@@ -0,0 +1,173 @@
+# MIP40c3-SPX: GovAlpha Core Unit Budget - GOV-001
+
+## Preamble
+
+```
+MIP40c3-SP#: X
+Author(s): @LongForWisdom
+Contributors: Payton Rose (@prose11)
+Status: RFC
+Date Applied: 2021-05-12
+Date Ratified: N/A
+```
+
+## Sentence Summary
+
+MIP40c3-SPX adds the budget for Core Unit GOV-001: GovAlpha.
+
+## Paragraph Summary
+MIP40c3-SPX adds the budget for Core Unit GOV-001: GovAlpha. This budget proposal covers a total of 3 months (July, August, and September of 2021). It communicates three focuses for GovAlpha over the period: Delegation, Education, and Quality Assurance. GovAlpha is asking for a total of 532,750 DAI over 3 months - 382,750 DAI will cover operations while 150,000 DAI will remain in a GovAlpha administered multi-sig wallet to ensure emergency continuity. 
+
+### Motivation
+
+We are proposing this Core Unit Budget modification because GovAlpha requires an operating budget in order to be able to fulfill its mandate.
```

Comment:
Change this to: "This Core Unit Budget modification is being proposed as GovAlpha requires an operating budget for the fulfillment of its mandate."

Reply:
We've gone with first person plural fairly consistently, so not sure changing it here is really a benefit or critical point. Passive voice gets a little tiresome for some people too.